connectors/source-http/README.md

```diff
@@ -99,6 +98,26 @@ Make sure the `target` value in your config file is `http://localhost:31081` so
 
 Open a terminal and use the following command to send an http request to HTTP Source
 
+<<<<<<< HEAD
+=======
```

Comment:
it occurs because of a code conflict, when you resolve it, please delete the no use code